src/lib/y2network/connection_config/base.rb

```diff
@@ -62,6 +64,15 @@ def type
         const_name = self.class.name.split("::").last.upcase
         InterfaceType.const_get(const_name)
       end
+
+      # @param interfaces [Y2Network::InterfacesCollection]
+      def update_interfaces!(interfaces)
+        nil
```

Comment:
ugh, what? intentions?

Reply:
Basically trying to look the best place and naming for replacing interface names by objects in master / slaves connections or vlan for example.

Reply:
i currently know of only one usage of this so putting it to generic ancestor makes no sense to me.